packages/babel-plugin-transform-for-of/test/fixtures/opt/array-binding/output.js

```diff
@@ -1,11 +1,13 @@
 const x = [];
+var _arr = x;
```

Comment:
Is it possible to move this inside the `for` head?

Reply:
Yes i actually did that, but it changes a lot of tests. Would do it separate.